grader-notebook/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM illumidesk/illumidesk-notebook:aec555e49be6
```

Comment:
@jgwerner 
I had to change this static tag to the latest built tag of illumidesk-notebook/Dockerfile image for the tests to pass as the previous image didn't have our async_nbgrader package

Reply:
@rupeshparab sounds good, we should probably default to latest anyway for dev and then we can set the image:tag version explicitly for our own deployments.